rpcs3/evdev_joystick_handler.cpp

```diff
@@ -168,6 +168,30 @@ void evdev_joystick_handler::Close()
     }
 }
 
+inline int Deadzone(f32 input)
```

Comment:
Deadzone should be lowercase, to match with most of the function naming scheme in the file.